### ISSUE TITLE
WrongTypeError exception when get_followers() is invoked

### ISSUE THREAD

**Issue by ap-Codkelden** [MarshalX/atproto#128]
When I try to get user's followers with the given code:

```
from atproto import Client, models

client = Client()
client.login('some_handle', 'app_password')

params = models.AppBskyGraphGetFollowers.Params(actor='some_actor')
client.bsky.graph.get_followers(params=params)
```

I get the following error:

    atproto.exceptions.WrongTypeError: wrong value type for field "followers.viewer.mutedByList.purpose" - should be "Literal" instead of value "app.bsky.graph.defs#modlist" of type "str"

The underlying cause of this error is `config.check_types` procedure:

```
dacite.exceptions.WrongTypeError: wrong value type for field "followers.viewer.mutedByList.purpose" - should be "Literal" instead of value "app.bsky.graph.defs#modlist" of type "str"
```
If type checking (lines 68-69 [here](https://github.com/konradhalas/dacite/blob/10a9ec40fc5874ae434aa68b975d1b1bf667a42f/dacite/core.py#L68)) is commented out, list of followers is successfully received.

`atproto` version is 0.0.24

**Comment by MarshalX** [comment]
hi, im working on migration from dacite to pydantic v2. the progress goes well. Please check the dev version of the sdk. Is the bug still relevant? https://github.com/MarshalX/atproto/pull/127

**Comment by MarshalX** [comment]
hm. looks like to reproduce I should mute some accounts by list first

**Comment by MarshalX** [comment]
reproduced, thank you! 

```
pydantic_core._pydantic_core.ValidationError: 1 validation error for Response
follows.1.viewer.mutedByList.purpose
  Input should be 'Modlist' [type=literal_error, input_value='app.bsky.graph.defs#modlist', input_type=str]
```

**Comment by ap-Codkelden** [comment]
Got the same error with `atproto@pydantic-models`.

**Comment by MarshalX** [comment]
fixed in c3815b22a8793c19ad67b8eaa905d940f74897e9

**Comment by MarshalX** [comment]
unit tests was added in 0393a959434d8a98848f6c30e554cf2fbc1db5eb
